.github/workflows/check-milestone-label.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     if: ${{ github.actor != 'dependabot[bot]' }}  # dependabot prs do not have to be in milestone
     steps:
       - name: require milestone label for pull request
-        uses: docker://agilepathway/pull-request-label-checker:latest
+        uses: docker://agilepathway/pull-request-label-checker:latest@sha256:14f766a581beb1c2a43448b8cdc360f810f39101137c87dd55d252bb1a2e85dc
```

Comment:
Do you need to remove the tag?

Reply:
IIRC the digest takes precedence 🤔

Reply:
I approved but will defer to you for merging

Reply:
OK, proceed with merging.